EmployeesSalaries/EmployeesSalaries/Services/EmployeeSalaryService.cs

```diff
@@ -0,0 +1,26 @@
+﻿using EmployeesSalaries.Models;
+using EmployeesSalaries.Models.Interfaces;
+
+namespace EmployeesSalaries.Services
+{
+    public class EmployeeSalaryService : IEmployeeSalaryService
+    {
+        List<IEmployeeSalaryCalculater> salaryCalculaters = new List<IEmployeeSalaryCalculater> {
+            new ManagerSalary(),
+            new SalesSalary(),
+            new LeadDevSalary(),
+            new HRSalary(),
+            new DevSalary()
+        };
+        public double CalcSalary(IEmployee employee)
+        {
+            IEmployeeSalaryCalculater calc = GetCalculater(employee);
+            return calc.GetTotalSalary();
+        }
+        private IEmployeeSalaryCalculater GetCalculater(IEmployee employee) // gets the correct calculater from employee type
+        {
+            return salaryCalculaters
```

Comment:
why do you use DevSalary in Employee service, and why it's the default one?

Reply:
It's used in Employee Salary Service
It didn't make since